plugins/out_cloudwatch_logs/cloudwatch_logs.c

```diff
@@ -391,14 +391,8 @@ static void cb_cloudwatch_flush(const void *data, size_t bytes,
 
     ctx->buf->put_events_calls = 0;
 
-    if (ctx->create_group == FLB_TRUE && ctx->group_created == FLB_FALSE) {
-        ret = create_log_group(ctx);
```

Comment:
`ret` unused now. Causing tests to fail.

Reply:
Thanks